backend/src/main/java/com/woowacourse/levellog/team/domain/Team.java

```diff
@@ -68,10 +69,10 @@ private void validatePlace(final String place) {
 
     private void validateStartAt(final LocalDateTime startAt) {
         if (startAt == null) {
-            throw new InvalidFieldException("시작 시간이 없습니다. 입력한 시작 시간 : [null]");
+            throw new InterviewTimeException("입력한 시작 시간 : [null]", "시작 시간이 없습니다.");
         }
         if (LocalDateTime.now().isAfter(startAt)) {
-            throw new InvalidFieldException("잘못된 시작 시간을 입력했습니다. 입력한 시작 시간 : [" + startAt + "]");
+            throw new InterviewTimeException("잘못된 시작 시간을 입력했습니다. 입력한 시작 시간 : [" + startAt + "]");
```

Comment:
저라면 아래처럼 자세하게 적을 것 같아요. 잘못된 시작 시간이 어떻게 잘못된지 클라이언트에서도 알아차리기 쉬우니까요 😊
```suggestion
            throw new InterviewTimeException("시작 시간은 현재 시간 이후여야 합니다. 입력한 시작 시간 : [" + startAt + "]");
```